### ISSUE TITLE
Signature verification failed

### ISSUE THREAD

**Issue by smival** [myflashlab/inAppPayments-ANE#23]
Hi,
tryed to use demo ANE via alpha test mechanism in google play.
I got code for all steps from your demo.
- After call Billing.doPayment got this error:
"Signature verification failed for sku no_ad : response - 1003"

if I repeat doPayment, got this:

"This item is already purchased..."

- After app restart my onInitHandler got this:
"Error refreshing inventory (querying owned items) response - 003 Purchase signature
verification failed"

For Billing.init() function I am using licence key from "Services and API" section
of google dev console. "MIIBIjANBg..." Test devide profile does't equals my publisher account.
So whats wrong?
ps: I use debug mode when init the ANE

**Comment by myflashlab** [comment]
These error messages usually happen if there's something wrong in your setup. please look for similar problems in already closed issues and you might find an answer there. moreover, check [here](http://stackoverflow.com/questions/14600664/android-in-app-purchase-signature-verification-failed) with a similar error message as yours.

but before everything, make sure you have gone through this checklist: https://github.com/myflashlab/inAppPayments-ANE/blob/master/FD/src/MainFinal.as#L220-L238

**Comment by myflashlab** [comment]
Please update this ticket about how it goes?

**Comment by smival** [comment]
hi, I tryed following steps
- remove code Billing.IS_DEBUG_MODE = true; 
- upload new build to alpha testing
- after about hour google inform me about the app has been updated
- on a test device after open new build I still see old error "Error refreshing inventory (querying owned items) response - 003 Purchase signature verification failed". Looks like I cant do any purchases while this error is here. Can I clear all purchases data for user? Or need I register new user in google play to avoid ane init error due last failed purchase?
ps: also my app fits all steps you provided by link above.
